History.md

```diff
@@ -1,3 +1,7 @@
+### 2.3.1 - *July 20 2017*
+
+  * Fix [https://github.com/web-animations/web-animations-js/issues/157](missing web-animations.min.js issue)
```

Comment:
Mmm, the instructions should probably say to push to origin/dev to avoid this problem in future.